docs/mddocs/Quickstart/ollama_portable_zip_quickstart.zh-CN.md

```diff
@@ -223,7 +224,7 @@ Ollama 默认从 Ollama 库下载模型。通过在**运行 Ollama 之前**设
 
 ### 签名验证
 
-针对 2.2.0 版本的 portable zip/tgz, 可以使用如下命令验证其签名：
```

Comment:
No need to change for signature verification. As only 2.2.0 has signature for now